jsk_fetch_robot/README.md

```diff
@@ -196,3 +196,35 @@ FAQ
 ```
 
   You might be forget to `source setup.bash` before you run `roseus`
+
+Administration
+--------------
+
+- set global environment variables https://github.com/jsk-ros-pkg/jsk_robot/issues/859#issuecomment-341269420
+```
+$ cat /etc/profile.d/jsk.sh 
+# added by furushchev (2017.11.2)
+export ROSLAUNCH_SSH_UNKNOWN=1                 # enable to run roslaunch with " Server' not found in known_hosts" 
+export JSK_DATA_CACHE_DIR=/etc/ros/jsk_data    # store recognition data within common directories to reduce hdd usage
+```
+
```

Comment:
@k-okada The first user who has `sudo` may have to create parent directory and give a right permission: `sudo mkdir /etc/ros/jsk_data && sudo chmod 0777 /etc/ros/jsk_data`. (Sorry I forgot to add this procedure in the issue)

Reply:
Also please note that scripts located in `/etc/profile.d` will be enabled on next login to the shell, so it is necessary to first log out and re-login to apply this change to current users.